adaptor-config.properties.sample

```diff
@@ -54,6 +54,21 @@ documentum.src =
 # are pushed to GSA.
 # documentum.pushLocalGroupsOnly = false
 
+# Custom query to send modified documents. Incremental updates uses this query
+# to find the modified documents and send to GSA. {0} will be replaced by the
+# modified date and {1} will be replaced by the object ID. The select list
+# needs to include r_modify_date_str, r_object_id, i_chronicle_id
+# and object_name.
+# documentum.updatedDocumentsQuery = SELECT r_modify_date, r_object_id, \
+#     i_chronicle_id, object_name, DATETOSTRING_LOCAL(r_modify_date, \
+#     ‘yyyy-mm-dd hh:mi:ss') AS r_modify_date_str \
```

Comment:
Done. Changed it. I got the actual query from runtime and replaced where necessary though.